hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -707,17 +707,26 @@ public SnapshotDiffReport getSnapshotDiffReport(Path snapshotDir,
       if (takeTemporaryToSnapshot || takeTemporaryFromSnapshot) {
         OFSPath snapPath = new OFSPath(snapshotDir.toString(), config);
         if (takeTemporaryToSnapshot) {
-          objectStore.deleteSnapshot(snapPath.getVolumeName(),
-              snapPath.getBucketName(), toSnapshot);
+          deleteSnapshot(toSnapshot, snapPath);
         }
         if (takeTemporaryFromSnapshot) {
-          objectStore.deleteSnapshot(snapPath.getVolumeName(),
-              snapPath.getBucketName(), fromSnapshot);
+          deleteSnapshot(fromSnapshot, snapPath);
         }
       }
     }
   }
 
+  private void deleteSnapshot(String snapshot, OFSPath snapPath) {
+    try {
+      objectStore.deleteSnapshot(snapPath.getVolumeName(),
+          snapPath.getBucketName(), snapshot);
+    } catch (IOException exception) {
+      LOG.warn("Failed to delete the temp snapshot with name {} in bucket"
+              + " {} and volume {} after snapDiff op.", snapshot,
+          snapPath.getBucketName(), snapPath.getVolumeName());
```

Comment:
I think we should log the exception message(``exception.getMessage()``) also, else it would be very difficult to debug why the delete failed